pyproject.toml

```diff
@@ -23,8 +23,11 @@ requests-oauthlib="1.3.0"
 six="1.15.0"
 urllib3="1.25.9"
 Werkzeug="1.0.1"
+gunicorn = "^20.0.4"
 
 [tool.poetry.dev-dependencies]
+pytest="5.4.3"
+selenium="3.141.0"
```

Comment:
Have you been adding to this file manually?
You can just run `poetry add selenium --dev`